base-theme/assets/js/sentry.ts

```diff
@@ -1,10 +1,11 @@
 import * as Sentry from "@sentry/browser"
 
-export const initSentry = () => {
+export const initSentry = (): typeof Sentry => {
+  const sentryDSN = process.env.SENTRY_DSN
   Sentry.init({
-    release: RELEASE_VERSION,
-    dsn:
-      "https://eee58f41dda54d2b814296e12dced4b7@o48788.ingest.sentry.io/5304953"
+    release:    RELEASE_VERSION,
+    dsn:        sentryDSN,
+    beforeSend: event => (sentryDSN ? event : null)
```

Comment:
I can confirm trying it without the `beforeSend` is fine. Node renders this as `var sentryDSN = "MISSING_ENV_VAR".SENTRY_DSN;` which would make me think that it would throw a connection error or something, but even after removing the `beforeSend` I didn't have any errors in my console.

Reply:
beforesend removed. Please have a look